extensions/components/com_redcore/admin/views/webservice/tmpl/default.php

```diff
@@ -14,150 +14,12 @@
 JHtml::_('behavior.keepalive');
 JHtml::_('rbootstrap.tooltip');
 JHtml::_('rjquery.chosen', 'select');
+JHtml::script(JUri::root() . '/media/redcore/js/redcore.js');
```

Comment:
I know gulp is supposed to automagiclly condense the JS file, but it doesn't seem to be doing it. Do I need to add something to the gulp config?